README.md

```diff
@@ -164,6 +166,95 @@ geographic region in support of yourproject’s security and compliance needs.
 See [Running in particular zone/region doc](docs/running_in_particular_zone_region.md).
 
 
+## Running VCF files to BigQuery Preprocessor
```

Comment:
Could you make "Running jobs in a particular region/zone" with `##` and put it at the end? it's common for any job users want to run.